cmake/oneflow.cmake

```diff
@@ -209,7 +209,10 @@ add_custom_target(of_format
   COMMAND ${Python_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/ci/check/run_clang_format.py --source_dir ${CMAKE_CURRENT_SOURCE_DIR}/oneflow --fix --quiet
   COMMAND ${Python_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/ci/check/run_py_format.py --source_dir ${CMAKE_CURRENT_SOURCE_DIR} --fix
   )
-
+# clang tidy
+add_custom_target(of_tidy
+  COMMAND ${Python_EXECUTABLE} ${CMAKE_SOURCE_DIR}/ci/check/run_clang_tidy.py --source_dir ${CMAKE_SOURCE_DIR} --build_dir ${CMAKE_BINARY_DIR}
```

Comment:
Maybe you need to refer here: https://github.com/PragmaTwice/oneflow/blob/ci-clang-tidy-add-more-checkers/.github/workflows/simple.yml#L65